src/api.ts

```diff
@@ -0,0 +1,24 @@
+const URL_USERS = 'https://jsonplaceholder.typicode.com/users';
+const URL_TODO = 'https://jsonplaceholder.typicode.com/todos';
+
+const getUsersFromServer = () => {
+  return fetch(URL_USERS)
+    .then(data => data.json());
+};
+
+const getTasksFromServer = () => {
+  return fetch(URL_TODO)
+    .then(data => data.json());
+};
+
+export const fullTaskList = async () => {
```

Comment:
```suggestion
export const getFullTaskList = async () => {
```
don't forget, this is function, not a variable with static value